### PR TITLE
Enable html in templates

### PR DESCRIPTION
**Pull request by athulkrishnaaei**
added create_link helper

https://github.com/ayrat555/el_monitorro/issues/98

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -31,4 +31,4 @@ url = "2"
 rayon = "1.5"
 
 [dev-dependencies]
-mockito = "0.31"
\ No newline at end of file
+mockito = "0.31"
```
Comment (ayrat555) [inline Cargo.toml:34]: let's remove this change

File: Cargo.lock [hunk 2]
```diff
@@ -171,9 +171,9 @@ checksum = "5827cebf4670468b8772dd191856768aedcb1b0278a04f989f7766351917b9dc"
 
 [[package]]
 name = "cpufeatures"
-version = "0.2.3"
+version = "0.2.4"
```
Comment (ayrat555) [inline Cargo.lock:174]: let's remove changes in this file

Comment (athulkrishnaaei) [reply Cargo.lock:174]: its automatically genearating  version = "0.2.4" when running the bot

File: Cargo.toml (hunk 1 above)
Comment (athulkrishnaaei) [reply Cargo.toml:34]: should i remove this line from the code its in the new version of el_monitorro

**Pull request merged**